internal/characters/amber/bunny.go

```diff
@@ -108,12 +236,11 @@ func (c *char) overloadExplode() {
 		if atk.Info.AttackTag != attacks.AttackTagOverloadDamage {
 			return false
 		}
+		c.bunnies[0].ae.Info.Abil = manualExplosionAbil
 
 		for _, v := range c.bunnies {
-			c.bunnies[0].ae.Info.Abil = manualExplosionAbil
-			c.Core.QueueAttackEvent(&v.ae, 1)
+			v.Kill()
 		}
```

Comment:
Please check whether the current bunny is within the overload attack pattern via `v.IsWithinArea(atk.Pattern)` before exploding it. If the bunny is out of range, it shouldn't explode.

Reply:
Is this supposed to explode the bunnies if it's the burst that triggers the OL?

Reply:
No, just when the CA causes overloaded. Just checked ingame.

Reply:
How do I do the `v.IsWithinArea(atk.Pattern)` , the atk given by the OL subscription is the triggering atk. Do I just do my own  `combat.NewCircleHitOnTarget(target, nil, 3)`?